dll/win32/shell32/shellmenu/CMenuFocusManager.cpp

```diff
@@ -657,6 +657,17 @@ LRESULT CMenuFocusManager::GetMsgHook(INT nCode, WPARAM hookWParam, LPARAM hookL
                 msg->lParam = 0;
                 msg->wParam = 0;
             }
+            else
+            {
+                if (msg->message == WM_SYSKEYDOWN &&
+                    m_current->type == MenuBarEntry &&
+                    (('A' <= msg->wParam && msg->wParam <= 'Z') ||
+                     ('0' <= msg->wParam && msg->wParam <= '9'))
```

Comment:
```suggestion
                     ('0' <= msg->wParam && msg->wParam <= '9')))
```
Do we miss a closing brace here? 3 levels of opening, 2 levels of closing as far as I can see/count.

Reply:
resolved